lib/src/rules/unnecessary_lambdas.dart

```diff
@@ -168,6 +168,12 @@ class _Visitor extends SimpleAstVisitor<void> {
         rule.reportLint(nodeToLint);
       }
     } else if (node is MethodInvocation) {
+      var target = node.target;
+      if (target is SimpleIdentifier &&
+          target?.staticElement is PrefixElement) {
```

Comment:
Should all prefixed imports be ignored? Or just those that are deferred? For example I could have

```dart
import 'package:quiver/strings.dart' as strings;

int countBlanks(List<String> list) {
  return list.where((s) => strings.isBlank(s)).length;
}
```

This can (and should) be linted? `return list.where(strings.isBlank).length;`

And a nit: No need for the null-aware operator. If we get past the is check, it won't be null anymore.

Reply:
Good catch(es).  Thanks!

Reply:
https://github.com/dart-lang/linter/pull/2178